cg/apps/crunchy/crunchy.py

```diff
@@ -135,11 +141,15 @@ def is_fastq_compression_done(compression_obj: CompressionData) -> bool:
             return False
         LOG.info("SPRING metadata file found")
 
-        crunchy_metadata: CrunchyMetadata = get_crunchy_metadata(
-            compression_obj.spring_metadata_path
-        )
+        try:
+            crunchy_metadata: CrunchyMetadata = files.get_crunchy_metadata(
+                compression_obj.spring_metadata_path
+            )
+        except ValidationError:
```

Comment:
I think it would be nice to not except this here! Knowing where validation fails can be helpful. It came in handy when hunting a bug last week

Reply:
Ok I think you are right

Reply:
Changed to fail hard